core/core.js

```diff
@@ -219,6 +219,14 @@ valuePropertyDescriptor.value = function specialize(prototypeProperties, constru
             Montage.defineProperty(constructor, "blueprintModuleId", prototypeProperties.blueprintModuleId);
         }
```

Comment:
OK - I think this was required for backwards compatibility.  Not sure how to fix this one.